src/messenger.rs

```diff
@@ -79,3 +80,7 @@ pub fn start_messenger(
         }
     }
 }
+
+pub fn get_new_conn_id() -> Uuid {
```

Comment:
Is this more of a server concern? I don't feel too strongly either way

Reply:
Oops forgot to remove that from previous tests, thanks for pointing it out